OS2Forms/os2forms8
# Merging develop into master to add os2forms_forloeb_profile and Docksal environment update

This PR includes following changes:

- The addition of a new install profile for OS2forms med Forløb
- Updates of Docksal environment
- Additional documentation for development environment and deployment
- Inclusion of os2forms_forloeb module in composer.json
- A variety of changes to themes from Bellcom additions to the develop branch
- All Maestro and webform templates from 3rd sprint

Test procedure with Docksal:

1. Clone the git repository and checkout this PR
   ```sh
   git clone git@github.com:OS2Forms/os2forms8.git
   ```

2. Enter the newly created project directory
   ```sh
   cd os2forms8
   ```

3. Start docksal environment
   ```sh
   fin start
   ```

4. Create local settings
   ```sh
   cp web/sites/example.settings.local.php web/sites/default/settings.local.php
   ```

5. Add file permission fix to settings.local.php. See https://docs.docksal.io/apps/drupal/#file-permissions
   ```php
   // web/sites/default/settings.local.php

   $settings['file_chmod_directory'] = 0777;
   $settings['file_chmod_file'] = 0666;
   ```

6. Install dependencies and enable modules
   ```sh
   fin build-forloeb
   ```

7. Configure trusted hosts in settings.local.php (add the following if not present)
   ```php
   // web/sites/default/settings.local.php

   $settings['trusted_host_patterns'] = ['^os2forms8.docksal$', '^localhost$'];
   ```
You should now be able to browse to the application at `http://os2forms8.docksal`
